CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **Features**:
 
 - Add error and sample rate fields to the replay event parser. ([#1745](https://github.com/getsentry/relay/pull/1745))
+- Add SSL support to `relay-redis` crate. ([#1772](https://github.com/getsentry/relay/pull/1772))
```

Comment:
Does this mean connecting to TLS now works out of the box, by using the [`rediss://`](https://github.com/redis-rs/redis-rs#tls-support) scheme?

Reply:
Yep. If the relay compiled with `ssl`  feature it should be possible to use `rediss` scheme with TLS support.

Reply:
@olksdr It doesn't look like Relay is compiled with the `ssl` feature on public docker images. Is it possible to add that capability?

Reply:
@SmoshySmosh  it looks like we do build the images with `ssl` enabled. Could you, please, open the issue in this repo with the error you get and the config you have? 
Thanks!